.github/workflows/publish.yml

```diff
@@ -4,14 +4,10 @@ on:
   push:
     branches: # Snapshots
       - main
-    tags: ["*"] # Releases
+    tags: ['**'] # Releases
 
 jobs:
   publish-artifacts:
     name: Publish / Artifacts
     uses: playframework/.github/.github/workflows/publish.yml@v2
-    secrets:
-      username: ${{ secrets.SONATYPE_USERNAME }}
-      password: ${{ secrets.SONATYPE_PASSWORD }}
-      pgp_passphrase: ${{ secrets.PGP_PASSPHRASE }}
-      pgp_secret: ${{ secrets.PGP_SECRET }}
+    secrets: inherit
```

Comment:
@ihostage You probably like this new feature 😉 
* https://github.blog/changelog/2022-05-03-github-actions-simplify-using-secrets-with-reusable-workflows/
* https://docs.github.com/en/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow

Reply:
Yeeees, cooool! 🕺